source/atomics.tex

```diff
@@ -798,7 +798,7 @@
 
 \pnum
 The implementation shall provide the function template specializations
```

Comment:
Yes.

In any case, these provisions are mostly vacuous. We have general wording in the library front matter that says exactly that: If the standard shows a signature, the implementation is required to provide a definition for it. I'd also appreciate a cleanup of the funny "integral-type" descriptions. I'm not totally clear when the text here prescribes a non-template overload or a (full) specialization of a function template. But that's for another day.